pkg/controller/vitesscluster/reconcile_vtadmin.go

```diff
@@ -341,12 +341,12 @@ func (r *ReconcileVitessCluster) createWebConfigSecret(ctx context.Context, vt *
 	// Variables to hold the key, value and secret name to use
 	configKey := vtadmin.WebConfigFileName
 	configVal := fmt.Sprintf(`window.env = {
-     'REACT_APP_VTADMIN_API_ADDRESS': "%s",
-     'REACT_APP_FETCH_CREDENTIALS': "omit",
-     'REACT_APP_ENABLE_EXPERIMENTAL_TABLET_DEBUG_VARS': false,
-     'REACT_APP_BUGSNAG_API_KEY': "",
-     'REACT_APP_DOCUMENT_TITLE': "",
-     'REACT_APP_READONLY_MODE': %s,
```

Comment:
We should probably keep the old env variables for one release to ensure backward compatibility.
We can drop these after 2.10.0